package.json

```diff
@@ -57,7 +57,13 @@
     "grunt-prompt": "~1.3.0",
     "grunt-replace": "0.6.2",
     "grunt-sass": "~1.1.0-beta",
+    "gulp": "3.8.10",
+    "gulp-iconfont": "1.0.0",
+    "gulp-iconfont-css": "0.0.9",
+    "gulp-install": "0.2.0",
+    "gulp-sketch": "0.0.7",
     "jquery": "2.1.4",
+    "jsdoc": "^3.4.0",
```

Comment:
Compare the output of the two output generators:
jsdoc: ![screen shot 2015-11-23 at 18 02 59](https://cloud.githubusercontent.com/assets/978461/11343498/9ad64e9c-920c-11e5-8181-292e19be26ae.png)
---
documentation.js: ![screen shot 2015-11-23 at 18 03 41](https://cloud.githubusercontent.com/assets/978461/11343499/9aeff144-920c-11e5-90c0-699b8301d8ca.png)
---

IMHO the 2nd (documentation.js) yields better result, primary because:
- it's more concise and legible output (at least by default?)
- it also has a search filter to easily find classes/methods without having to scroll or search inline